wasp/src/main/java/com/orhanobut/wasp/MethodInfo.java

```diff
@@ -64,6 +66,12 @@ private void parseMethodAnnotations() {
                 continue;
             }
 
+            if (annotationType == RetryPolicy.class) {
+                final RetryPolicy rp = ((RetryPolicy) annotation);
```

Comment:
I would suggest to not use final keyword in local variables unless there is a need to such using in anonymous class, of course the usage is up to developer but I find that it makes less readability. although we don't gain too much things by using it. I also would suggest to not use abbrevations, instead I prefer one word abbrevation such "policy" instead of "retryPolicy" for local variables. If the line is too long, I use the group wrapping style.

<code>
RetryPolicy policy =  ((RetryPolicy) annotation);
retryPolicy = new WaspRetryPolicy(
         policy.initialTimeout(), policy.maxNumRetries(), policy.backoffMultiplier()
);
</code>